streams/jvm/src/main/scala/zio/stream/experimental/platform.scala

```diff
@@ -503,3 +503,43 @@ trait ZStreamPlatformSpecificConstructors {
 
   trait ZStreamConstructorPlatformSpecific extends ZStreamConstructorLowPriority1
 }
+
+trait ZSinkPlatformSpecificConstructors {
+  self: ZSink.type =>
+
+  /**
+   * Uses the provided `OutputStream` to create a [[ZSink]] that consumes byte chunks
+   * and writes them to the `OutputStream`. The sink will yield the count of bytes written.
+   *
+   * The caller of this function is responsible for closing the `OutputStream`.
+   */
+  final def fromOutputStream(
+    os: OutputStream
+  )(implicit trace: ZTraceElement): ZSink[Any, IOException, Byte, IOException, Byte, Long] = fromOutputStreamManaged(
```

Comment:
I hope it's correct to have IOException for both `InErr` and `OutErr` types.

Reply:
I think we need a review of the whole `ZSink` regarding this because what we probably want here is `InErr >: IOException` but `foldLeftChunksZIO` forces you to have the same in both type params now.

So let's merge it like this and we will do that review separately